thprac/src/thprac/thprac_gui_locale.cpp

```diff
@@ -474,7 +474,7 @@ static ImWchar baseUnicodeRanges[] =
         if (__glocale_disabled) {
             if (__glocale_disabled == 7) {
                 MessageBoxW(nullptr, L"No font can be loaded.\nthprac will now terminate.", L"Fatal error", MB_OK | MB_ICONERROR);
-                TerminateProcess(GetCurrentProcess(), -1);
+                TerminateProcess(GetCurrentProcess(), ERROR_FILE_CORRUPT);
```

Comment:
I think it'd be better to fix any warnings caused by `TerminateProcess(GetCurrentProcess(), ...)` lines by either changing it to `ExitProcess` or (if possible), returning back to WinMain, which then returns, right there in the same commit. I wanna get rid of all the  `TerminateProcess(GetCurrentProcess(), ...)` lines anyways cause that's jank